src/openforms/analytics_tools/models.py

```diff
@@ -300,7 +323,11 @@ def is_google_analytics_enabled(self) -> bool:
 
     @property
     def is_govmetric_enabled(self) -> bool:
-        return self.govmetric_source_id and self.enable_govmetric_analytics
+        required_fields_present = (
+            self.govmetric_source_id_form_finished
+            and self.govmetric_source_id_form_aborted
+        )
+        return required_fields_present and self.enable_govmetric_analytics
```

Comment:
```suggestion
        return (
            self.govmetric_source_id_form_finished
            and self.govmetric_source_id_form_aborted
            and self.enable_govmetric_analytics
        )
```
So that it stays consistent with the other properties